web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -1248,7 +1248,7 @@
     "wfsIndexingRefreshError": "An error occurred when refreshing the list:",
     "wfsIndexingError": "Error encountered",
     "fromRegistry": "From registry",
-    "registry-help": "<a href='http://inspire.ec.europa.eu/registry'>Re3gistry software</a> is a tool to publish registers. Registers provide a means to assign identifiers to items and their labels, definitions and descriptions (in different languages). The INSPIRE registry provides a central access point to a number of centrally managed INSPIRE registers.",
+    "registry-help": "<a href='http://inspire.ec.europa.eu/registry'>Registry software</a> is a tool to publish registers. Registers provide a means to assign identifiers to items and their labels, definitions and descriptions (in different languages). The INSPIRE registry provides a central access point to a number of centrally managed INSPIRE registers.",
```

Comment:
The original text was fine, the tool is call `Re3gistry`: https://joinup.ec.europa.eu/solution/re3gistry, please update the PR and will merge it, thanks.

Reply:
Done, sorry!